crates/forge/tests/integration/setup_fork.rs

```diff
@@ -259,7 +259,7 @@ fn fork_get_block_info_fails() {
             #[test]
             #[fork(url: "{INTEGRATION_RPC_URL}", block_id: BlockId::Number(999999999999))]
             fn fork_get_block_info_fails() {{
-                starknet::get_block_info().unbox();
+                let _bx_i = starknet::get_block_info().unbox();
```

Comment:
Why?

Reply:
I did it because of `must_use` + `unused_variable` but i guess i can leave it boxed